protocol/dubbo/config.go

```diff
@@ -90,6 +90,57 @@ type (
 	}
 )
 
+func GetDefaultClientConfig() ClientConfig {
+	return ClientConfig{
+		ReconnectInterval: 0,
+		ConnectionNum:     16,
+		HeartbeatPeriod:   "5s",
+		SessionTimeout:    "20s",
+		PoolSize:          64,
+		PoolTTL:           600,
+		GrPoolSize:        1200,
+		QueueLen:          64,
+		QueueNumber:       60,
+		GettySessionParam: GettySessionParam{
+			CompressEncoding: false,
+			TcpNoDelay:       true,
+			TcpKeepAlive:     true,
+			KeepAlivePeriod:  "120s",
+			TcpRBufSize:      262144,
+			TcpWBufSize:      65536,
+			PkgWQSize:        512,
+			TcpReadTimeout:   "1s",
+			TcpWriteTimeout:  "5s",
+			WaitTimeout:      "1s",
+			MaxMsgLen:        2014,
```

Comment:
why use 2014?  and I think 2014 is too short.
I use 102400

Reply:
fix